Mlem/Views/Tabs/Inbox/Inbox View Logic.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  Inbox Feed View Logic.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2023-06-26.
+//
+
+import Foundation
+
+extension InboxView {
+    func refreshFeed() async {
+        do {
+            try await mentionsTracker.refresh(account: account)
+            try await messagesTracker.refresh(account: account)
+            try await repliesTracker.refresh(account: account)
+            
+            aggregateAllTrackers()
+        } catch APIClientError.networking {
+            // TODO: we're seeing a number of SSL related errors on some instances while loading pages from the feed
```

Comment:
Nit:

should we remove the commented sections here and leave TODOs to add error handling specific for the inbox view?